bundle/deploy/terraform/plan.go

```diff
@@ -32,7 +78,12 @@ func (p *plan) Apply(ctx context.Context, b *bundle.Bundle) error {
 		return fmt.Errorf("terraform not initialized")
 	}
 
-	cmdio.LogString(ctx, "Starting plan computation")
+	if p.goal == PlanDeploy {
+		cmdio.LogString(ctx, "Planning deployment")
```

Comment:
This doesn't seem relevant to me as a user?

Reply:
This was inspired by terraform where we can have planning take a significant amount of time so having this in would be an indication that there's some progress happening in the background.

I am fine with being convervative here initially, because for bundles we do not anticipate the planning time to be particularly long anytime soon. Removed these logs.